From 2eac836c4085beb679b665c0c7cfd3f49b928230 Mon Sep 17 00:00:00 2001
From: Andreas Sturmlechner <andreas.sturmlechner@gmail.com>
Date: Tue, 9 Oct 2018 16:03:12 +0200
Subject: [PATCH] openconnect: add missing dependency Qt5Xml

Summary: QDomDocument is imported from vpn/openconnect/openconnectauth.cpp

Test Plan: Build succeeds now.

Reviewers: #plasma

Subscribers: plasma-devel

Tags: #plasma

Differential Revision: https://phabricator.kde.org/D16073
---
 vpn/openconnect/CMakeLists.txt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/vpn/openconnect/CMakeLists.txt b/vpn/openconnect/CMakeLists.txt
index cf43740d..c3f07088 100644
--- a/vpn/openconnect/CMakeLists.txt
+++ b/vpn/openconnect/CMakeLists.txt
@@ -6,6 +6,9 @@ set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_MODULE_PATH})
 
 find_package(OpenConnect ${MINIMUM_OPENCONNECT_VERSION_REQUIRED} MODULE)
 if (OPENCONNECT_FOUND)
+
+    find_package(Qt5Xml ${QT_MIN_VERSION} CONFIG REQUIRED)
+
     if (${OPENCONNECT_VERSION} VERSION_LESS "3.99")
 #         macro_optional_find_package(OpenSSL)
 #         macro_log_feature(OpenSSL_FOUND "OpenSSL headers" "Encryption suite" "http://www.openssl.org" FALSE "" "Needed for OpenConnect support in Network Management")
@@ -49,6 +52,7 @@ if (OPENCONNECT_FOUND)
             KF5::IconThemes
             KF5::KIOWidgets
             KF5::CoreAddons
+            Qt5::Xml
             ${OPENCONNECT_LIBRARIES}
             ${maybe_OPENSSL_LIBRARIES}
         )
-- 
2.19.1

